vuepress/docs/next/docs/reference/backup-restore.md

```diff
@@ -0,0 +1,88 @@
+
+## General Guidelines for Bundles
+Bundles should be reinstalled when restoring from a backup with the strategy provided by the bundle creator. Below are some suggested guidelines. 
+
+1. Export all the EntandoDeBundle custom resources from the source instance and restore them to the new instance.
+Then manually install each bundle. The reinstallation will create the proper client credentials and permissions required by the bundle.
+2. If custom roles and permissions were applied in the original instance, manually transfer these as well. Note, if a different Keycloak is used for the restored instance, then the client and permissions for the bundle may need to be exported/imported to the new instance. See the [Backing Up and Restoring Keycloak](../../tutorials/devops/backing-restoring-keycloak.md) tutorial for details.
+3. For Bundle DB schemas, different methods are required for auto-provisioned and external databases.
+
+    A. For auto-provisiond DBMS schemas:  
+             1. When the provisioning is complete, scale down the plugin deployment to 0. Restore the source data with the database-native backup/restore tools and then scale up the deployment.  
+             2. In case of GCG-generated Spring Boot plugins, the plugin's application will autonomously apply the Liquibase data migration scripts.   
+             3. In order to connect to an internal auto-provisioned DBMS, use kubectl port-forward against the bundle DB.   
+
+    B. For External DBMS schemas:  
+       1. Export all the EntandoDeBundle CRs from the source instance and restore them to the target instance.  
+       2. Then create the restored instance DB schema by making a copy of the source instance DB schema with the DB native backup/restore tools.  
+       3. Copy DB credential secrets from the original instance into the restored instance.  
+       4. Reinstall the bundles.   
+
+
+### Private Repository Bundles
+If a bundle originated from a private repository in the original instance, the secrets, configurations, etc. will need to be replicated in the new instance. 
+* For private Docker registry bundles, see [Install Bundle from a Private Image Registry](https://developer.entando.com/v7.1/tutorials/curate/bundle-private-images.html) tutorial. 
+* For bundles from a private Git repository, see [Install Bundles from a Private Git Repository](https://developer.entando.com/v7.1/tutorials/curate/private-git-repo.html)  tutorial.
+
+## Configure the Service Address
+If the restored instance service address will remain the same as the original, and needs to be on the same cluster, the two ingresses can collide and end with failure for the restored instance.
```

Comment:
no comma before "and needs..." these two things both must be true and there is no natural pause

i don't think an ingress is **on** a cluster??

end with -> result in 

"If the restored instance service address will remain the same as the original and **will apply to** the same cluster, the two ingresses can collide and **result in the** failure **of** the restored instance." (or "cause the restored instance to fail")